.travis.yml

```diff
@@ -0,0 +1,34 @@
+sudo: true
+dist: trusty
+
+language: php
+
+services:
+  - mongodb
+
+php:
+  - hhvm-3.12
+  - hhvm-3.15
+
+matrix:
+  allow_failures:
+    - php: hhvm-3.12
+    - php: hhvm-3.15
```

Comment:
If you're ignoring the entire matrix, why ignore anything at all? Just let the builds fail.